scala/scala-impl/src/org/jetbrains/plugins/scala/project/template/Downloader.scala

```diff
@@ -71,13 +72,18 @@ object Downloader {
       "updateClassifiers")
   }
 
-  private def sbtCommandsForHydra(platform: Platform, version: String) = Seq(
-    s"""set scalaVersion := "${version.split("_")(0)}"""",
-    s"""set credentials := Seq(Credentials("Artifactory Realm", "repo.triplequote.com", "${HydraCredentialsManager.getLogin}", "${HydraCredentialsManager.getPlainPassword}"))""",
-    s"""set resolvers := Seq(Resolver.url("Triplequote Plugins Ivy Releases", url("https://repo.triplequote.com/artifactory/ivy-releases/"))(Resolver.ivyStylePatterns), Resolver.url("Triplequote sbt-plugin-relseases", url("https://repo.triplequote.com/artifactory/sbt-plugins-release/"))(Resolver.ivyStylePatterns),  "Triplequote Plugins Releases" at "https://repo.triplequote.com/artifactory/libs-release-local/")""",
-    s"""set libraryDependencies := Seq("com.triplequote" % "hydra_${version.split("_")(0)}" % "${version.split("_")(1)}", ("com.triplequote" % "hydra-bridge_1_0" % "${version.split("_")(1)}").sources())""",
-    "updateClassifiers",
-    "show dependencyClasspath")
+  private def sbtCommandsForHydra(platform: Platform, version: String) = {
+    val settings = HydraApplicationSettings.getInstance()
+    val repositoryName = settings.getHydraRepositoryName
+    val repositoryUrl = settings.getHydraRepositoryUrl
```

Comment:
`repositoryUrl` should be a URL (if you can't change this in the settings class, then just call `new URL` here).